src/quo/components/list/item.cljs

```diff
@@ -70,29 +70,31 @@
         children))
 
 (defn icon-column
-  [{:keys [icon icon-bg-color icon-color size icon-container-style]}]
+  [{:keys [icon icon-bg-color icon-color icon-size icon-container-style]}]
```

Comment:
hmm, imo this prop name doesn't need to be changed or is there a particular reason for it?

Reply:
Changed.